### ISSUE TITLE
Addon to survey 2 loot system: Dimensional/Global split loot tables

### ISSUE THREAD

**Issue by zalfenior** [Tslat/Advent-Of-Ascension#124]
The proposed (and likely to be implemented) new loot system also includes a global loot table

I propose that this system is extended so that every dimension gets their own version of the loot table with exclusive items. Current foraging system has a very weak version of this for the level 65 drops. 
example of the global portion
Copper coins
Metal slug
Holly arrows
Bones (basic material list here, as I am unsure what the plan is for that)
Rosite nuggets/ingots (or new repair item)
Ration - New item, basically a saturating quick to eat cracker, modeled after military rations

Now the new items would effectively be themed around the individual dimensions and would include basic materials already in the mod, crops and potentially other useful items. The point of the crops would be to allow people to make semi permanent bases within their favored dimensions. This might be an entirely different suggestion though
Basically we have 1 or 2 slots for ore related items, 1 or 2 for crops and a few slots for thematically relevant, useful items. Naturally, dimensions without ores wouldnt need to drop those and dimensions like Vox Ponds (naturally inhospitable to life) and crystevia (lol whats a plant?) would not need crops

for example:
Overworld: Tea seeds (due to how useful they are), Iron ingots/nuggets etc
Nether: Nether wart and Spice flower (New item, used to brew fire resistance tea) and gold ingots/nuggets or nether quartz
Google doc link is below as I intend on making a list for all reasonable dimensions
[Relevant google doc](https://docs.google.com/document/d/1H4b8dESEg_2iEg-vgBbucaya3mBVVlEBZ4nK3s2hKUU/edit?usp=sharing)

**Comment by Immortallis1** [comment]
I like the ideas and they feel real enough and also add utility. Of course some tweaking would be needed but items like fashioned claw, the light sources, and food items are nice.
